app/assets/stylesheets/layout/_footer.scss

```diff
@@ -0,0 +1,3 @@
+.background-footer{
+  background-color: var(--black-background)!important;
```

Comment:
the way to call your color in scss should be like this:
- `$black-background;`
- Remove the important.